docs/RELEASE.md

```diff
@@ -0,0 +1,21 @@
+# Release
+
+Instruction on how to release new package
+
+## Steps
+
+### Bump Version
+
+1. Run `./scripts/release-version-bump <RELEASE_TYPE>` to update `package.json`.
+   This will also create and push to a new branch.
+
+1. Create PR
```

Comment:
should this be `2.`?

Reply:
markdown auto render the number correctly.  It's bad to explicitly state the number, b/c then when you add/remove new item, you have to modify all subsequent value